be/src/vec/aggregate_functions/aggregate_function_stddev.h

```diff
@@ -200,13 +192,71 @@ struct PopData : Data {
     }
 };
 
+template <typename Data>
+struct StddevName : Data {
+    static const char* name() { return "stddev"; }
+};
+
+template <typename Data>
+struct VarianceName : Data {
+    static const char* name() { return "variance"; }
+};
+
+template <typename Data>
+struct VarianceSampName : Data {
+    static const char* name() { return "variance_samp"; }
+};
+
+template <typename Data>
+struct StddevSampName : Data {
+    static const char* name() { return "stddev_samp"; }
+};
+
+template <typename Data, bool is_nullable = false>
+class AggregateFunctionPop final
```

Comment:
You should learn more abstract templates

`AggregateFunctionPop` and  `AggregateFunctionSamp`